code/iaas/logic/src/main/resources/META-INF/cattle/core-process/spring-core-processes-context.xml

```diff
@@ -18,15 +18,15 @@
         <process:defaultProcesses resourceType="agent" />
 
         The above XML is the equivalent as doing the below, 
-        refer to cattle-process-common/src/main/resources/META-INF/cattle/process/defaults.properties for the definitive list
+        refer to cattle/code/iaas/logic-common/src/main/resources/META-INF/cattle/process/defaults.properties for the definitive list
 
-        TODO: This is out of date
-
-        <process:process name="agent.create" resourceType="agent" start="requested" transitioning="registering" done="inactive" />
+        <process:process name="agent.create" resourceType="agent" start="requested" transitioning="registering" done="inactive" delegate="agent.activate" />
         <process:process name="agent.activate" resourceType="agent" start="inactive" transitioning="activating" done="active" />
```

Comment:
Actually, did you miss the registering as a start state for activate?
https://github.com/rancherio/cattle/blob/master/code/iaas/logic-common/src/main/resources/META-INF/cattle/process/defaults.properties#L3